Developer-Guide.md

```diff
@@ -342,7 +342,9 @@ $ curl -L https://raw.githubusercontent.com/kata-containers/packaging/master/ker
 $ make ARCH=${kernel_dir} -j$(nproc)
 $ kata_kernel_dir="/usr/share/kata-containers"
 $ kata_vmlinuz="${kata_kernel_dir}/kata-vmlinuz-${kernel_version}.container"
-$ [ $kernel_arch = ppc64le ] && kernel_file="$(realpath ./vmlinux)" || kernel_file="$(realpath arch/${kernel_arch}/boot/bzImage)"
+$ [ $kernel_arch = ppc64le ] && kernel_file="$(realpath ./vmlinux)" \
+|| [ $kernel_arch = aarch64] && kernel_file="$(realpath arch/arm64/boot/Image)" \
```

Comment:
A couple of things:

- Can you not fold the lines using `\`'s - github renders long lines with scroll bars so they are not necessary.

- This is incorrect as there is no space before `]`. It should be:
  ```
  || [ $kernel_arch = aarch64 ] && kernel_file="$(realpath arch/arm64/boot/Image)"
  ```